go/vt/wrangler/vexec.go

```diff
@@ -346,7 +347,10 @@ type ReplicationStatusResult struct {
 	TargetLocation ReplicationLocation
 	// MaxVReplicationLag represents the maximum vreplication lag seen across all shards.
 	MaxVReplicationLag int64
-
+	// MaxVReplicationTransactionLag represents the lag between the current time and the timestamp of the last transaction seen across all shards.
```

Comment:
last replicated transaction timestamp OR heartbeat timestamp (if there have been no writes to replicate from the source), right?

Reply:
updated as per your suggestion